llvm/test/tools/llvm-mca/RISCV/SpacemitX60/atomic.s

```diff
@@ -0,0 +1,312 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+# RUN: llvm-mca -mtriple=riscv64 -mattr=+rva22u64 -mcpu=spacemit-x60 -iterations=1 < %s | FileCheck %s
```

Comment:
I don't think you need `-mattr` here, the CPU definition should already specify all the extensions